sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -180,6 +188,9 @@ export class StudentService extends RecordDataModelService<Student> {
         student.sinValidation = sinValidation;
         await entityManager.getRepository(Student).save(student);
       }
+
+      await this.importSFASOverawards(student, studentSIN, entityManager);
```

Comment:
For this scenario, I would prefer that the method parameters are restricted to what is really needed, in this case, only `lastName` and `birthDate` instead of the entire student object.